tests/saw/sike/LICENSE.TXT

```diff
@@ -0,0 +1,4 @@
+SIKE saw verification
```

Comment:
This needs to include the full apache 2.0 license from https://aws.amazon.com/apache-2-0/